src/pages/index.tsx

```diff
@@ -238,422 +134,534 @@ export const getStaticProps = (async ({ locale }) => {
+                Svg={Svg}
+                href={href}
+                label={label}
+                className={cn("xl:hidden", className)}
+                variant="col"
+              >
+                <p className="text-body">{description}</p>
+              </SvgButtonLink>
+              <SvgButtonLink
+                Svg={Svg}
+                href={href}
+                label={label}
+                className={cn("hidden xl:block", className)}
+                variant="row"
+              >
+                <p className="text-body">{description}</p>
+              </SvgButtonLink>
+            </Fragment>
+          ))}
+        </div>
+
+        {/* Use Cases - A new way to use the internet */}
+        <Section
+          id="use"
+          className={cn(
+            "max-lg:-mx-4 max-lg:flex max-lg:w-[100vw] max-lg:flex-col max-lg:overflow-hidden max-lg:px-4 sm:max-lg:-mx-6 sm:max-lg:px-6", // Mobile: Swiper cards
+            "lg:grid lg:grid-cols-bento lg:gap-4" // Desktop: BentoBox grid
+          )}
+        >
+          <div
+            className={cn(
+              "flex flex-col",
+              "lg:col-span-12 xl:col-span-3 xl:col-start-2"
+            )}
           >
-            <Box
-              flex="0 0 50%"
-              maxW={{ lg: "75%" }}
-              p={{ sm: 8, lg: 24 }}
-              boxSize="full"
-            >
-              <SectionHeading fontFamily="inherit" mb={6}>
-                <Translation id="page-index:page-index-get-started" />
-              </SectionHeading>
-              <SectionDecription>
-                <Translation id="page-index:page-index-get-started-description" />
-              </SectionDecription>
-            </Box>
-            <ImageContainer>
-              <Image
-                src={hackathon}
-                alt={t("page-index:page-index-get-started-image-alt")}
-                width={720}
-                backgroundSize="cover"
-                background="no-repeat 50px"
-              />
-            </ImageContainer>
-          </Flex>
-          <CardContainer minChildWidth={{ lg: "480px" }}>
-            {cards.map((card, idx) => (
-              <StyledActionCard
-                key={idx}
-                boxShadow={cardBoxShadow}
-                m={0}
-                title={card.title}
-                description={card.description}
-                alt={card.alt}
-                href={card.href}
-                image={card.image}
-                imageWidth={320}
+            <div className="w-fit rounded-full bg-primary-low-contrast px-4 py-0 text-sm uppercase text-primary">
+              {t("common:nav-use-cases-label")}
+            </div>
+            <h2 className="mb-4 me-4 mt-2 text-5xl font-black xl:mb-6 xl:text-7xl">
+              {t("page-index:page-index-bento-header")}
+            </h2>
+          </div>
+
+          {/* Mobile */}
+          <Swiper
+            options={{ effect: "cards", createElements: true }}
```

Comment:
Is `createElements` necessary? I removed it and haven't seen any clear change, everything seems to be working normally.

Asking because I see this error on the console
![image](https://github.com/user-attachments/assets/676be74e-18ca-42b8-968c-eb6ecfd1322b)

Reply:
Yeah, I think you're right that we can remove that... I had added it previously to debug, but it seems to have been unrelated. Will remove.